### ISSUE TITLE
[Question] kindness mode: serving much more clients than a browser extension. Intentional?

### ISSUE THREAD

**Issue by WofWca** [guardianproject/orbot#1105]
**Describe Your Question**
I have been using both the Snowflake extension and Orbot's "Kindness mode". I notice that Orbot is getting a lot more clients per week. It's in the ballpark 50, whereas the extension is getting like one per day or less. And they are on the same network (WiFi), running 24 / 7 (except when I take the phone outside, but I doubt this is the main factor).
I know this is most likely mostly determined by the polling interval of standalone Snowflake and browser extension Snowflake. See https://gitlab.torproject.org/tpo/anti-censorship/pluggable-transports/snowflake-webext/-/issues/43#note_2820375
I.e. standalone Snowflake is coded to get much more clients per unit of time that the extension. But I think it shouldn't be the case for mobile phones.

The question is: is this something that has been considered? If so, why has it been decided to give mobile devices more clients than to PC browsers?

**Is Your Question Related?**
No

**Additional Context**

**Comment by tladesignz** [comment]
No, this is not intentional, but accidental. I guess we should tune down the poll interval at least when the device is running on battery.

**Comment by WofWca** [comment]
As was said in https://github.com/tladesignz/IPtProxy/pull/58#issuecomment-2527607585, this should be addressed in this repo.

**Comment by bitmold** [comment]
Closing this as I'm going through the backlog now. FWIW in `SnowflakeProxyWrapper` we set `Capacity` to 1. Maybe this could be parameterized in the new UI, @tladesignz 

```kt
  with(proxy) {
                this?.proxyTypeIdentifier = "orbot-android"
                this?.brokerUrl = fronts["snowflake-target-direct"]
                this?.capacity = 1L
                // ...
}
```

**Comment by WofWca** [comment]
Yep, sounds good. And `PollInterval` also has effect on how often you get a client, but looks like this has been addressed in https://github.com/guardianproject/orbot-android/pull/1242 already.
